src/reference/asciidoc/quick-tour.adoc

```diff
@@ -122,7 +122,7 @@ private Map<String, Object> senderProps() {
 ----
 ====
 
-==== With Java Configuration
+==== With Spring Configuration in Java
```

Comment:
It's not clear why this is needed; it's certainly not a typo. "Spring" is implied since this is a Spring project.

Reply:
sure, The example followed is using spring configuration, so I thought it be a more relevant title.

Reply:
Well, we definitely talk in this document only about Spring. If the story is about something else, we are more specific.

It is really called `Java Configuration` in Spring world: https://docs.spring.io/spring/docs/current/spring-framework-reference/core.html#beans-java

I don't think we are going to change the text flow.

So, sorry, but closed as `Invalid`. There is just no typo in this sentence.